disk_areas.md

```diff
@@ -39,10 +41,13 @@ For more details you can look at: [Managing Data on Abel] and more generally at
 <img src="https://docs.google.com/drawings/d/e/2PACX-1vSY_KCj3fubTH1zk6ZkOL6eLhoOOuAbp4bfu1YkOAvkadHhPfbuZrsepwHCUpEqwr45Zqt2hlEoCwVk/pub?w=960&amp;h=720">
 
 **SLURM** is the queue system and scheduler daemon (program/process which always runs in the background on Abel). It allows optimizing computing resources and schedule when jobs are to be run on $SCRATCH.- **All long jobs should be sumbited to to the queue system SLURM (and are run on $SCRATCH)**. 
+> There are 2 ways to use slurm SLURM queue system: 1. **interactively** with the command: `qlogin` and 2. by **submitting a script**. 
+> - when you use `qlogin` you actually are logged on a computing node that now executes commands from the shell. Usage: small tests
```

Comment:
Usage: small tests, to test out a working process, or if you have interactive programs.

Reply:
ok - added